app/mailers/host_mailer.rb

```diff
@@ -39,7 +39,8 @@ def summary(options = {})
   def error_state(report)
     report = Report.find(report)
     host = report.host
-    owners = host.owner.recipients_for(:puppet_error_state)
+    owners = host.owner.recipients_for(:puppet_error_state) if host.owner.present?
+    owners ||= []
     raise ::Foreman::Exception.new(N_("unable to find recipients")) if owners.empty?
```

Comment:
@ohadlevy I think @unorthodoxgeek is right here, it'll still raise in the situations he said.